Containerfile.helpers

```diff
@@ -0,0 +1,28 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-builder-multi-openshift-4.18 AS fetcher
+WORKDIR /oc
+RUN <<EOF
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+if [[ "$(uname -m)" == "x86_64" ]]; then
+  curl -L "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux.tar.gz" | tar xz
+fi
+if [[ "$(uname -m)" == "aarch64" ]]; then
+  curl -L "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux-arm64.tar.gz" | tar xz
```

Comment:
thought (non-blocking): In the future, we can remove this for CI use-cases because the OpenShift CI system has a way to inject the `oc` command into any task where it is needed.

To be clear: This comment is only here for posterity.